### ISSUE TITLE
cluster submit should exit with same exit code as gcloud

### ISSUE THREAD

**Issue by danking** [Nealelab/cloudtools#72]
```
(hail) dking@wmb16-359 # cluster submit foo foo
Submitting to cluster 'foo'...
gcloud command:
gcloud dataproc jobs submit pyspark foo \
    --cluster=foo \
    --files= \
    --properties=
ERROR: (gcloud.dataproc.jobs.submit.pyspark) NOT_FOUND: Not found: Cluster projects/broad-ctsa/regions/global/clusters/foo
(hail) dking@wmb16-359 # $(exit 3)
(hail) 3 dking@wmb16-359 # 
```
My shell prompt prints non-zero exit codes before my username. Note that `cluster` exited 0.

**Comment by danking** [comment]
`start` also appears to have this problem.

**Comment by danking** [comment]
resolved by #74
